modules/hcp-aws-create-role-sts-creds.adoc

```diff
@@ -17,7 +17,13 @@ Before creating a hosted cluster on {aws-first}, you must create an {aws-short}
 $ aws sts get-caller-identity --query "Arn"
 ----
 +
-Note the ARN of your user to use in the next step.
+.Example output
+[source,terminal]
+----
+arn:aws:iam::1234567890:user/<aws_username>
```

Comment:
if we just want to get the arn without double quotations, I think the command should be 
`$ aws sts get-caller-identity --query "Arn" --output text`

e.g.
$ aws sts get-caller-identity --query "Arn"
"arn:aws:iam::***:user/heli"
$ aws sts get-caller-identity --query "Arn" --output text
arn:aws:iam::***:user/heli

Reply:
Thanks! I added `--output text`.